docs/src/usage/std.md

```diff
@@ -78,6 +78,24 @@ Example:
       type: any
 ```
 
+#### "must_use"
+This field is used for checking if the return value of a function is used.
+
+- `true` - The return value of this function must be used.
+- `false` - The default. The return value of this function does not need to be used.
+
+Example:
+```yml
+table.find:
+  args:
+    - type: table
+      observes: read
+      required: true
+    - type: any
+      required: true
+  must_use: true  # The return value of this function must be used
```

Comment:
See the existing spec for `table.find`. Let's keep the example consistent and correct.
https://github.com/Kampfkarren/selene/blob/65174d1f056ce71442a34a9aeb9ed9c968c82b96/selene-lib/default_std/luau.yml#L376-L382

Reply:
Resolved in: https://github.com/Kampfkarren/selene/pull/583/commits/9dd838a2143095670586564e7c15a5434578eff4